opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/authorization/consent/xs2a-authorize-consent.bpmn20.xml

```diff
@@ -117,26 +117,52 @@
     <sequenceFlow id="sid-7ac9f81f-cc30-4c83-83a4-05c93e63fe70" name="Aspsp started Authorization" sourceRef="sid-28d87700-36b4-4ef7-9f81-42473767bb7e" targetRef="scaChallengeSelector">
       <conditionExpression xsi:type="tFormalExpression">#{xs2aConsentInfo.isAuthorisationStarted(CONTEXT)}</conditionExpression>
     </sequenceFlow>
+    <exclusiveGateway id="isUserPinRequired" name="is Pin Required" default="noPinRequired"/>
+    <sequenceFlow id="userPinRequired" name="Pin Required" sourceRef="isUserPinRequired" targetRef="sid-35d13776-ce27-462c-805f-f6e4200710cb">
+      <conditionExpression xsi:type="tFormalExpression">#{xs2aConsentInfo.isStartConsentAuthorizationWithPin(CONTEXT)}</conditionExpression>
+    </sequenceFlow>
+    <exclusiveGateway id="sid-35d13776-ce27-462c-805f-f6e4200710cb" name="Password/OTA present?" default="pinNotAvaialable"/>
+    <sequenceFlow id="pinNotAvaialable" name="No password available" sourceRef="sid-35d13776-ce27-462c-805f-f6e4200710cb" targetRef="askUserForpin"/>
+    <serviceTask id="askUserForpin" name="Ask user to input password. Suspends to wait" flowable:async="true" flowable:exclusive="true" flowable:delegateExpression="${xs2aAskForPassword}" flowable:triggerable="true" isForCompensation="false"/>
+    <serviceTask id="startAuthorizationWithPin" name="Authenticate user" flowable:async="true" flowable:exclusive="true" flowable:delegateExpression="${xs2aStartConsentAuthorizationWithPin}">
```

Comment:
It is wrong place for `startAuthorizationWithPin`. It should be here:
![image](https://user-images.githubusercontent.com/18463491/134637987-e29e66a5-c28b-4696-a95b-a6371ea6bb14.png)